hockeypuck_rock/delete_keys.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import argparse
+import logging
+import os
+
+import psycopg2
+
+logger = logging.getLogger(__name__)
+
+
+def get_db_connection():
+    """Connect to the Postgres database."""
+    db_password = os.getenv("POSTGRESQL_DB_PASSWORD")
+    db_name = os.getenv("POSTGRESQL_DB_NAME")
+    db_host = os.getenv("POSTGRESQL_DB_HOSTNAME")
+    db_user = os.getenv("POSTGRESQL_DB_USERNAME")
+    dsn = f"dbname={db_name} user={db_user} password={db_password} host={db_host}"
+    try:
+        conn = psycopg2.connect(dsn)
+        conn.autocommit = True
+        return conn
+    except Exception as e:
```

Comment:
You should avoid catching generic exceptions, as you might catch something unexpected